lib/src/api/event/player/cast_time_updated_event.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:bitmovin_player/src/api/event/event.dart';
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'cast_time_updated_event.g.dart';
+
+/// Emitted when the time update from the currently used
+/// cast-compatible device is received.
+@JsonSerializable(explicitToJson: true)
+class CastTimeUpdatedEvent extends Event with EquatableMixin {
```

Comment:
Generally I agree that it could make sense. More favourable would be a seamless experience, where there is no need for special cast events IMO. There is the `TimeChangedEvent` which is emitted for playback time changes also during casting. But the duration is not covered by other events during casting I think.

I will revisit this and let you know.

Reply:
I checked the ReactNative player and there we also did not expose the `currentTime` or `duration` on the `CastTimeUpdatedEvent`.
Do you think we should keep this aligned?

Reply:
Yes, fine for me to keep it like this 👍